docs/development/architecture.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Architecture
+sidebar_position: 0
+---
+
+
+## Introduction
+
+Headlamp is a user-friendly and extensible Kubernetes UI designed to provide a seamless experience for managing Kubernetes clusters. It can connect to one or multiple Kubernetes clusters and can be deployed as a web app in-cluster or as a desktop app. Headlamp can be customized and 
```

Comment:
Unintended trailing sentence?

Reply:
Maybe mention what this section is about instead of mentioning what Headlamp is (because the intro to Headlamp should is given in a different section of the docs)?